Update pgpverify-maven-plugin configuration.

pstackle opened the pull request:
Improves the configuration of the pgpverify-maven-plugin to allow it to be re-enabled.

Defines plugin configuration to use both the file from the
`org.simplify4u:pgp-keys-map` dependency and a custom keys-map file to allow for
customizations when the `pgp-keys-map` project (https://github.com/s4u/pgp-keys-map/blob/master/resources/pgp-keys-map.list) is missing something.

Related: #10112

pstackle committed:
Update pgpverify-maven-plugin configuration.

Define plugin configuration to use both the file from the
org.simplify4u:pgp-keys-map dependency and a custom keys-map file to allow for
customizations when the pgp-keys-map project is missing something.

joschi commented on .mvn/pgpverify/custom-keys-map.list:
```diff
@@ -0,0 +1,10 @@
+# This fills in the gaps for https://github.com/s4u/pgp-keys-map
+
+# io.smallrye:jandex:jar:3.2.0 PGP key has been revoked and public key is not available
+io.smallrye:jandex:3.2.0    = !0x94976E17E18DD3201447286954963C3E875A56AE
```
```suggestion
io.smallrye:jandex:jar:3.2.0    = !0x94976E17E18DD3201447286954963C3E875A56AE
```

joschi committed:
Try fully-qualified artifact name for Jandex

```
[INFO] Receive key: https://keys.openpgp.org/pks/lookup?op=get&options=mr&search=0x94976E17E18DD3201447286954963C3E875A56AE
	to /home/runner/.m2/repository/pgpkeys-cache/94/97/94976E17E18DD3201447286954963C3E875A56AE.asc
Error:  Not allowed artifact io.smallrye:jandex:jar:3.2.0 and keyID:
	io.smallrye:jandex:3.2.0 = 0x94976E17E18DD3201447286954963C3E875A56AE
	https://keys.openpgp.org/pks/lookup?op=vindex&fingerprint=on&search=0x94976E17E18DD3201447286954963C3E875A56AE
```

joschi commented:
Addressed via https://github.com/dropwizard/dropwizard/pull/10998, https://github.com/dropwizard/dropwizard/pull/10999, and https://github.com/dropwizard/dropwizard/pull/11000.
